CONTRIBUTING.md

```diff
@@ -0,0 +1,216 @@
+# Contributing to win32-certstore
+
+We're glad you want to contribute to win32-certstore! The first
+step is the desire to improve the project.
+
+## Quick Contributing Steps
+
+1. Create an account on [GitHub](https://github.com).
+2. Create an account on the [Chef Supermarket](https://supermarket.chef.io/).
```

Comment:
This supermarket links are no more active please refer current Chef CONTRIBUTING.md to update this according to Chef standards